lib/activerecord-import/import.rb

```diff
@@ -332,8 +332,9 @@ def import!(*args)
     end
 
     def import_helper( *args )
-      options = { validate: true, timestamps: true, primary_key: primary_key }
+      options = { validate: true, timestamps: true }
       options.merge!( args.pop ) if args.last.is_a? Hash
+      options[:primary_key] = primary_key
```

Comment:
A comment might be a good idea here so that this doesn't get lost down the road.